### ISSUE TITLE
Reduce down time when restart ovs-ovn

### ISSUE THREAD

**Issue by oilbeater** [kubeovn/kube-ovn#2291]
https://github.com/ovn-org/ovn/commit/896adfd2d8b3369110e9618bd190d190105372a9
https://github.com/ovn-org/ovn/commit/fbb8e19313b59b7b05a7a7bd97900bd4c9ace694
https://github.com/ovn-org/ovn/commit/aa8c535a1faf4bd3c0494edce1e9dda8a0251263

**Comment by zhangzujian** [comment]
Network break was observed during ovs-ovn restart:

```shell
root@kube-ovn-control-plane:/# ping -DOq -i 0.001 10.16.0.8
PING 10.16.0.8 (10.16.0.8) 56(84) bytes of data.
[1675748969.534338] no answer yet for icmp_seq=22647
[1675748969.544743] no answer yet for icmp_seq=22648
[1675748969.796264] no answer yet for icmp_seq=22874
[1675748969.806656] no answer yet for icmp_seq=22875
^C
--- 10.16.0.8 ping statistics ---
27950 packets transmitted, 27946 received, 0.0143113% packet loss, time 28482ms
rtt min/avg/max/mdev = 0.059/0.113/11.985/0.119 ms
```

**Comment by zhangzujian** [comment]
Geneve interface was recreated during the ovs pod start up:

```shell
root@kube-ovn-worker2:/kube-ovn# ip -c a show genev_sys_6081
17: genev_sys_6081: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 65000 qdisc noqueue master ovs-system state UNKNOWN group default qlen 1000
    link/ether fa:1e:c7:a8:da:49 brd ff:ff:ff:ff:ff:ff
root@kube-ovn-worker2:/kube-ovn# ip monitor dev genev_sys_6081
17: genev_sys_6081: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 65000 qdisc noqueue state UNKNOWN group default
    link/ether fa:1e:c7:a8:da:49 brd ff:ff:ff:ff:ff:ff
17: genev_sys_6081: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 65000 qdisc noqueue state UNKNOWN group default
    link/ether fa:1e:c7:a8:da:49 brd ff:ff:ff:ff:ff:ff
17: genev_sys_6081: <BROADCAST,MULTICAST> mtu 65000 qdisc noqueue state DOWN group default
    link/ether fa:1e:c7:a8:da:49 brd ff:ff:ff:ff:ff:ff
Deleted 17: genev_sys_6081: <BROADCAST,MULTICAST> mtu 65000 qdisc noop state DOWN group default
    link/ether fa:1e:c7:a8:da:49 brd ff:ff:ff:ff:ff:ff
^C
root@kube-ovn-worker2:/kube-ovn# ip -c a show genev_sys_6081
18: genev_sys_6081: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 65000 qdisc noqueue master ovs-system state UNKNOWN group default qlen 1000
    link/ether 7a:b4:13:b8:c0:4f brd ff:ff:ff:ff:ff:ff
```

**Comment by zhangzujian** [comment]
```shell
pod-1:/kube-ovn# ping -DOq -i 0.001 10.16.0.7
PING 10.16.0.7 (10.16.0.7) 56(84) bytes of data.
[1676272884.469972] no answer yet for icmp_seq=20500
[1676272884.480685] no answer yet for icmp_seq=20501
[1676272884.490841] no answer yet for icmp_seq=20502
[1676272978.999356] no answer yet for icmp_seq=47402
[1676272979.011760] no answer yet for icmp_seq=47403
[1676272981.201277] no answer yet for icmp_seq=49530
```

ovs-vswitchd debug log:

```txt
# on the first node
2023-02-13T07:21:24.487Z|00056|tunnel|INFO|adding tunnel port ovn-9283e6-0 (::->172.18.0.2, key=flow, legacy_l2, dp port=3)
# on the second node
2023-02-13T07:22:59.007Z|00051|tunnel|INFO|adding tunnel port ovn-4ed288-0 (::->172.18.0.3, key=flow, legacy_l2, dp port=2)
```

**Comment by oilbeater** [comment]
Need underlay test

**Comment by zhangzujian** [comment]
# Underlay

## Within Node

```shell
pod-1:/kube-ovn# ping -DOq -i 0.001 172.18.0.10
PING 172.18.0.10 (172.18.0.10) 56(84) bytes of data.
[1676540814.850753] no answer yet for icmp_seq=28376
```

## Across Nodes

```shell
pod-1:/kube-ovn# ping -DOq -i 0.001 172.18.0.9
PING 172.18.0.9 (172.18.0.9) 56(84) bytes of data.
[1676540894.234955] no answer yet for icmp_seq=20582
```

**Comment by zhangzujian** [comment]
Tunnel interface recreation has been fixed:

[[ovs-dev] ofproto: Fix re-creation of tunnel backing interfaces on restart.](https://patchwork.ozlabs.org/project/openvswitch/patch/20230223163927.915879-1-i.maximets@ovn.org/)

Will try whether the patch works for ovs v2.16/v2.17.

**Comment by zhangzujian** [comment]
Tunnel interface recreation has been fixed in Kube-OVN release 1.9/1.10/1.11/master.
